sdk/servicebus/service-bus/test/utils/managementUtils.ts

```diff
@@ -0,0 +1,183 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+import {
+  QueueOptions,
+  TopicOptions,
+  SubscriptionOptions,
+  ServiceBusAtomManagementClient,
+  delay
+} from "../../src";
+
+import { EnvVarKeys, getEnvVars } from "./envVarUtils";
+
+const env = getEnvVars();
+let client: ServiceBusAtomManagementClient;
+
+async function getManagementClient() {
+  if (client == undefined) {
+    client = await new ServiceBusAtomManagementClient(env[EnvVarKeys.SERVICEBUS_CONNECTION_STRING]);
+  }
+  return client;
+}
+
+/**
+ * Utility to apply retries to `create` management operations.
+ * Will ignore if given entity is already existing.
+ * @param createEntityOperation
+ * @param checkIfEntityExistsOperation
+ */
+async function retryCreateManagementEntityOperation(
+  createEntityOperation: any,
+  checkIfEntityExistsOperation: any
+) {
+  const retryAttempts = 5;
+  const retryDelayInMs = 1000;
+
+  let succeeded: boolean = false;
+  let count = 0;
+  while (count < retryAttempts) {
+    try {
+      await createEntityOperation();
+      const entityExists = await checkIfEntityExistsOperation();
+      if (entityExists) {
```

Comment:
What is the need to check if entity exists if the create entity operation ran without any error?

Reply:
Based on testing, looks like delays were possible where the entity may not have been necessarily created. Similar for delete operations.